climada/hazard/centroids/centr.py

```diff
@@ -187,6 +194,21 @@ def from_base_grid(land=False, res_as=360, base_file=None):
             base_file = NATEARTH_CENTROIDS[res_as]
 
         centroids.read_hdf5(base_file)
+        if centroids.meta:
+            xres, xshear, xoff, yshear, yres, yoff = centroids.meta['transform'][:6]
+            shape = (centroids.meta['height'], centroids.meta['width'])
+            if yres > 0:
+                # make sure y-orientation is negative
+                centroids.meta['transform'] = rasterio.Affine(xres, xshear, xoff, yshear,
+                                                              -yres, yoff + (shape[0] - 1) * yres)
+                # flip y-axis in data arrays
+                for name in ["region_id", "dist_coast"]:
+                    if not hasattr(centroids, name):
+                        continue
+                    data = getattr(centroids, name)
+                    if data.size == 0:
+                        continue
+                    setattr(centroids, name, np.flipud(data.reshape(shape)).reshape(-1))
```

Comment:
Would it make sense to use centroids.check() before returning centroids? That would warn the user, if they assign a unknown file using the parameter base_file, and the resulting centroids are somehow not in order.

Reply:
Yes, good idea! I added a check.